package.json

```diff
@@ -3,13 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
-    "web-vitals": "^1.0.1"
+    "web-vitals": "^1.1.2"
   },
```

Comment:
- [ ] Please add `react-redux` in your dependencies

Reply:
Hello, thank you for the review. I did not understand your first request. Are you asking me to move my index.js file into the redux folder or you are asking me to create another index.js file?

![Screenshot from 2021-08-31 10-55-39](https://user-images.githubusercontent.com/64914462/131465137-1f509ad4-e64d-4f1b-a418-54ec831ba743.png)

The template above shows that I should have it in the src directory. 

Thank you for clarifying me